cmd/templates-api/main.go

```diff
@@ -48,7 +48,12 @@ func main() {
 
 	// Start DataDog tracer.
 	if envs.Get("DATADOG_ENABLED") != "false" {
-		tracer.Start(tracer.WithServiceName("templates-api"), tracer.WithLogger(ddLogger{logger}))
+		tracer.Start(
+			tracer.WithServiceName("templates-api"),
+			tracer.WithLogger(ddLogger{logger}),
+			tracer.WithRuntimeMetrics(),
+			tracer.WithAnalytics(true),
```

Comment:
Zapol som dalsie metriky, mozeme ich neskor vypnut, ak by to bolo zbytocne, ... nezaznamenal som ziaden rozdiel vo vykone.